tests/input_files/beam3r_line2_backweuler_browndyn_periodic_rve_dirich_crosslinking.dat

```diff
@@ -129,7 +129,7 @@ NODE 14 COORD  0.000000000000000e+00  4.000000000000000e+00  4.000000000000000e+
 1 BEAM3R LINE2 1  2 MAT 1 TRIADS  0 -1.57079632679 0 0 -1.57079632679 0
 2 BEAM3R LINE2 3  4 MAT 1 TRIADS  0 -1.57079632679 0 0 -1.57079632679 0
 3 BEAM3R LINE2 4  1 MAT 1 TRIADS  0 -1.57079632679 0 0 -1.57079632679 0
-----------------------------------------------------------PERIODIC BOUNDINGBOX ELEMENTS ELEMENTS
+----------------------------------------------------------PERIODIC BOUNDINGBOX ELEMENTS
```

Comment:
@isteinbrecher @davidrudlstorfer I found a mistake in a couple of beam input files. This was not caught before because the implementation inside `InputFile` had a subtle bug where certain section titles could have arbitrary trailing content.

Reply:
@sebproell thanks for the info - and thanks for fixing it along the way!